tests/prompt_strategies/conftest.py

```
@@ -180,6 +180,15 @@ def fixture_devstral_1_1_tokenizer():
     return tokenizer
 
 
+@pytest.fixture(name="qwen3_tokenizer")
+def qwen3_tokenizer_fixture(
+    download_qwen3_half_billion_model,
```

Ruff (0.13.1)

Unused function argument: `download_qwen3_half_billion_model`

(ARG001)